src/transport/net_ib.cc

```diff
@@ -131,6 +154,13 @@ NCCL_PARAM(IbEceEnable,"IB_ECE_ENABLE",1);
 NCCL_PARAM(IbDataDirect,"IB_DATA_DIRECT",1);
 NCCL_PARAM(IbQpsPerConn, "IB_QPS_PER_CONNECTION", 1);
 RCCL_PARAM(IbQpsPerP2p, "IB_QPS_PER_P2P", 0);
+NCCL_PARAM(IbGdrFlushDisable, "GDR_FLUSH_DISABLE", 0);
+
+// AMD AINIC
+RCCL_PARAM(CtsInlineData, "CTS_INLINE_DATA", -1);
+RCCL_PARAM(CtsOffloadEnabled, "CTS_OFFLOAD_ENABLED", -1);
```

Comment:
```suggestion
RCCL_PARAM(CtsInlineData, "CTS_INLINE_DATA", 1);
RCCL_PARAM(CtsOffloadEnabled, "CTS_OFFLOAD_ENABLED", 1);
```
Please refer my comment below.

Reply:
this will be taken care of and planned to handle differently.

Reply:
refactored code per discussion on 12/11/2025 and pushed updated to PR